### PR TITLE
refactor: Update dependencies to latest

### PR DESCRIPTION
**Pull request by dignifiedquire**

### CODE REVIEW

File: src/api/cat.js [hunk 1]
```diff
@@ -2,6 +2,6 @@
 
 const argCommand = require('../cmd-helpers').argCommand
 
-module.exports = send => {
+module.exports = (send) => {
```
Comment (diasdavid) [inline src/api/cat.js]: thank you :)

**Comment by diasdavid** [comment]
sweet, let just make sure travis and circle are happy

**Comment by dignifiedquire** [comment]
all happy

**Comment by diasdavid** [comment]
LGTM :)

**Pull request merged**